packages/babel-helpers/src/helpers.js

```diff
@@ -404,6 +405,30 @@ helpers.objectSpread = helper("7.0.0-beta.0")`
   }
 `;
 
+helpers.objectSpread2 = helper("7.0.0-beta.0")`
+  import defineProperty from "defineProperty";
+
+  export default function _objectSpread(target) {
```

Comment:
Should it be named objectSpread2? I wonder in the case someone uses babel-runtime, this might be a problem, but I don't know exactly how runtime works.

Reply:
Since this is a default export, the name doesn't matter.
Anyway, we should rename it for consistency.

Reply:
But where is the name in generated output coming from? Because right now it is called `_objectSpread` even with he new helper, which made me think it might collide in the runtime again.

Reply:
It does `generateUidIdentifier("objectSpread2")`, which removes `2` before generating the id.

Reply:
Would `objectSpreadTwo` is okay or should I fix the function to not remove `2`?

Reply:
It's ok as it is now for me.

Reply:
You mean this can be ignored?

>Anyway, we should rename it for consistency.

Reply:
We should rename the helper:
```js
export default function _objectSpread2
```
But is is not a problem if that `2` then gets automatically removed by `generateUid`.